webapp/src/Migrations/Version20210813195818.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20210813195818 extends AbstractMigration
+{
+    public function getDescription() : string
+    {
+        return 'Remove JudgehostRestriction data';
+    }
+
+    public function up(Schema $schema) : void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE judgehost DROP FOREIGN KEY judgehost_ibfk_1');
+        $this->addSql('DROP TABLE judgehost_restriction');
+        $this->addSql('DROP INDEX restrictionid ON judgehost');
```

Comment:
What is the “change polltime”?

Reply:
That should not be there, its part of the doctrine:migrations:diff wanting the change the whole DB.

Reply:
Our dev docs state how to fix this (set a car in .env.local)